djangocms_text_ckeditor/tests/test_field.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+from django.template import Context, Template
+
+from djangocms_helper.base_test import BaseTestCase
+
+from djangocms_text_ckeditor.test_app.forms import SimpleTextForm
+from djangocms_text_ckeditor.test_app.models import SimpleText
+
+
+class FieldTestCase(BaseTestCase):
+
+    text_normal = '<p>some non malicious text</p>'
+    text_with_iframe = ('<p>some non malicious text</p>'
+                        '<iframe src="http://www.w3schools.com"></iframe>')
+    text_with_script = ('<p>some non malicious text</p>'
+                        '<script>alert("Hello! I am an alert box!");</script>')
+
+    def test_model_field_text_is_safe(self):
+        original = 'Hello <h2>There</h2>'
+        template = Template("{{ obj.text }}")
+        text = SimpleText.objects.create(text='Hello <h2>There</h2>')
+        # Fetching a new instance should now have the string marked
+        # as safe.
+        text = SimpleText.objects.get(pk=text.pk)
+        rendered = template.render(Context({'obj': text}))
+        self.assertEqual(original, rendered)
+
+    def test_model_field_sanitized(self):
+        obj = SimpleText(text=self.text_normal)
+        obj.full_clean()
+        obj.save()
+
+        obj = SimpleText.objects.get(pk=obj.pk)
+
+        self.assertEqual(obj.text, self.text_normal)
+
+        obj = SimpleText(text=self.text_with_iframe)
+        obj.full_clean()
+        obj.save()
+
+        escaped = ('<p>some non malicious text</p>&lt;iframe '
```

Comment:
can this be placed as an instance attribute to reduce boilerplate code? (minor though)